modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/subcellularmodel/v0.1.2.json

```diff
@@ -35,10 +35,10 @@
               "description": "The synapse/ion channel release this model is part of.",
               "or": [
```

Comment:
This needs to be revised later. Modelling the membership of a model in a release should be handled differently.